### ISSUE TITLE
[Suggestion] Convert any oredict to any oredict?

### ISSUE THREAD

**Issue by mariodk** [WanionCane/UniDict#136]
hey im making a modpack that is simple/noob friendly and will have abit less ores/ingots & etc. than a standard modpack with the mods i have included

so i wanna ask if there is any way (or can be added in a update)
so i can turn everything "Osmium" into just use Silver oredict so every
Silver ingot, plate, ore, block & etc. is Osmium and visa versa

and then all Silver ingots will work on every repices/machines that needs Osmium Ingots(or other Osmium stuff) and visa versa 

same for any other metal / stuff with a oredict

and i found that doing this via CraftTweaker can be Very much annoying and not working as i wanted
since i have to "rename" all osmium items to silver version aswell as add silver oredict to osmium via crafttweaker
also found out if having   B:keepOneEntry=true in configs at same time giving silver oredict to osmium and visa versa via CraftTweaker
it crashes minecraft 

& the other metals that mods have added but i want to replace with others mainly thermalfoundation´s ores only so mekanism dont needs its own ore only used there and use Silver instead of Osmium & same for afew other mods )
or i have to change every recipes

so basically i want to merge Osmium stuff into Silver stuff
and make them both 1 and the same and unify them 
same with afew other things
even 3 or more metals: Osmium, aluminum and silver all the same and works between each other 

or a other example
if you want to go full vanilla metals/ores but still use mods that adds them 
you can do so Iron = Silver, Osmium, Aluminum
Gold = Platinum, 
and so on

also not just for merge 2 different metals into 1
a other thing i wanted to do: compressed "insert metal here" galacticraft = plates"insert metal here" since they are basically the same thing
